functions.yml

```diff
@@ -116,11 +122,12 @@ complex_type_manipulation:
         - name: struct
           doc: Struct from which to select fields
         - name: fields
-          doc: Fields to take
+          doc: Fields to select
           type: str
           is_var_args: true
+      returns: A struct containing only the indicated fields
     - name: vector_to_array
-      doc: Convert a spark.ml vector (sparse or dense) to an array of doubles
+      doc: Converts a ``spark.ml.linalg`` ``Vector`` (sparse or dense) to an array of doubles.
```

Comment:
Oh sorry, didn't realize we just haven't been writing out `linalg`. Can you remove that again? Sorry.

Reply:
Done